stdlib/public/Concurrency/SourceCompatibilityShims.swift

```diff
@@ -162,6 +162,15 @@ extension Task where Success == Never, Failure == Never {
       try await body(&group)
     }
   }
+
+  @available(*, deprecated, renamed: "Task.sleep(nanoseconds:)")
+  @_alwaysEmitIntoClient
+  public static func sleep(_ duration: UInt64) async {
```

Comment:
This should not be `@_alwaysEmitIntoClient` because it needs to remain in the ABI, and should therefore stay in `TaskSleep.swift`. `SourceCompatibilityShims.swift` should be able to be removed without any effect on the ABI.

Reply:
Removed `@_alwaysEmitIntoClient` and moved back to `TaskSleep.swift`.